### DISCUSSION TITLE
How to clear Recent Projects and Recent Files?

### DISCUSSION THREAD

**Discussion by ghost** [pascalabcnet/pascalabcnet#2948] [General]
There is nothing on the UI of the IDE to do this. I found the setting seems to be saved in PascalABCNET.ini, so it's very likely that I need to edit that file like when I need to configure the Font Size of the Text Editor to an odd value (the UI only shows even values). Even though it has the `.ini` extension, the file is indeed of type `xml`, a format that not designed to edit by hand.

**Comment by SunSerega** [comment]
> `xml`, a format that not designed to edit by hand

But XML is generally designed to be human-readable. And in many cases, it's written by hand and then read by a program. It's just that in this case the contents would easily break, with how it was written...
